src/_imaging.c

```diff
@@ -406,11 +406,11 @@ getlist(PyObject* arg, Py_ssize_t* length, const char* wrong_length, int type)
         // on this switch. And 3 fewer loops to copy/paste.
         switch (type) {
         case TYPE_UINT8:
-            itemp = PyInt_AsLong(op);
+            itemp = (int)PyInt_AsLong(op);
```

Comment:
Maybe it's better to declare itemp as long?
https://docs.python.org/3/c-api/long.html#c.PyLong_AsLong

Reply:
Thanks for your thoughts. See my previous comment about this being resolved.